msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -134,28 +131,26 @@ The JSON response body for `GetParticipant` API is:
 
 ### Response codes
 
-The `GetParticipant` API returns the following response codes:
+The following table provides the response codes:
 
 |Response code|Description|
 |---|---|
-| **403** | Get participant information isn't shared with the app. If the app isn't installed in the meeting, it triggers the most common error response 403. If the tenant admin disables or blocks the app during live site migration, 403 error response is triggered. |
+| **403** | Get participant information isn't shared with the app. If the app isn't installed in the meeting, it triggers the error response 403. If the tenant admin disables or blocks the app during live site migration, it triggers the error response 403. |
 | **200** | The participant information is successfully retrieved.|
 | **401** | The app responds with an invalid token.|
-| **404** | The meeting has either expired or participant cannot be found.|
+| **404** | The meeting has either expired or participants are not available.|
 
 ## NotificationSignal API
 
-All users in a meeting receive the notifications sent through the `NotificationSignal` API.
+All users in a meeting receive the notifications sent through the `NotificationSignal` API. `NotificationSignal` API enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send signal based on user action, an in-meeting dialog box. The API includes query parameter, examples, and response codes.
```

Comment:
Not sure what is being said here. 

Did you mean to say:
You can send a signal to the user by triggering an in-meeting dialog box?

Reply:
I have removed the sentence as per your previous comment.